MonoGame.Framework/GraphicsDeviceManager.cs

```diff
@@ -256,7 +256,7 @@ public void ApplyChanges()
 
             ((MonoGame.Framework.WinFormsGamePlatform)_game.Platform).ResetWindowBounds();
 
-#elif WINDOWS || LINUX
+#elif WINDOWS || DESKTOPGL
```

Comment:
I'm pretty sure this can just be `#elif DESKTOPGL` as `OpenTKGamePlatform` is only used on WindowsGL and Linux.